### PR TITLE
Add margin equal to strokeWidth to SVG export

### LINKED ISSUE #187: Add margin around output of SVG files
Lines with reasonable stroke width are cut off around the edges:

image

Adding a margin around the objects within the SVG file would be a quick fix. Alternatively, including the stroke width of lines in the total width of the graphical area would be ideal.

### PR DESCRIPTION
**Pull request by canadaduane**
Closes #187 

image

### CODE REVIEW

File: Graphics/Implicit/Export/PolylineFormats.hs [hunk 1]
```diff
@@ -24,9 +24,11 @@ import qualified Data.List as List
 
 svg :: [Polyline] -> Text
 svg plines = renderSvg . svg11 . svg' $ plines
-    where  
-      (xmin, xmax, ymin, ymax) = (minimum xs, maximum xs, minimum ys, maximum ys)
-           where (xs,ys) = unzip (concat plines)
+    where
+      strokeWidth = 1.0
+      (xmin, xmax, ymin, ymax) = ((minimum xs) - margin, (maximum xs) + margin, (minimum ys) - margin, (maximum ys) + margin)
+           where margin = strokeWidth / 2 + 0.1
```
Comment (julialongtin) [inline Graphics/Implicit/Export/PolylineFormats.hs]: 0.1 here is a magic number. will merge when it is removed.

**Comment by canadaduane** [comment]
There is a slight cropping of the edge of the stroke when I remove this 0.1
pad. Do you have a suggestion to keep the entire stroke visible that does
not include some kind of magic number?

**Comment by julialongtin** [comment]
can you send me some example files, so i can look at the output svg generated?

**Comment by canadaduane** [comment]
Here's an example of the issue, with and without an additional 0.1 margin:

implicitcad-svg-extra-margin

I'm using a simple `test.escad` file with contents `circle(30);`

**Comment by julialongtin** [comment]
I still think we should apply this patch, without the 0.1 modifier,
and file a different bug for either passing in a different modifier,
or calculating a modifer that guarantees spacing for the anti-aliased
pixels.

**Comment by canadaduane** [comment]
I don't have time to fix it correctly, but there you go.

**Comment by julialongtin** [comment]
Thanks!

**Pull request merged**